records_mover/db/bigquery/loader.py

```diff
@@ -121,19 +122,24 @@ def load(self,
 
         logger.info("Loading from records directory into BigQuery")
         # https://googleapis.github.io/google-cloud-python/latest/bigquery/usage/tables.html#creating-a-table
-        connection: Connection =\
-            self.db.engine.raw_connection().connection
+        connection: Connection = self.db.engine.raw_connection().connection
         # https://google-cloud.readthedocs.io/en/latest/bigquery/generated/google.cloud.bigquery.client.Client.html
         client: Client = connection._client
         project_id, dataset_id = self._parse_bigquery_schema_name(schema)
         job_config = self._load_job_config(load_plan)
 
+        try:
+            table_obj = client.get_table(f"{schema}.{table}")
+        except NotFound:
+            logger.error("BigQuery table %s.%s not found", schema, table)
+            raise
+
         all_urls = directory.manifest_entry_urls()
 
         job = client.load_table_from_uri(all_urls,
                                          f"{schema}.{table}",
                                          # Must match the destination dataset location.
-                                         location="US",
+                                         location=table_obj.location,
```

Comment:
This is the magic-- we just use the destination table's location. This seems to be what is mandated by BigQuery.